Funcky.Analyzers/Funcky.Analyzers/AnonymousFunctionMatching.cs

```diff
@@ -0,0 +1,25 @@
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Funcky.Analyzers;
+
+internal static class AnonymousFunctionMatching
+{
+    /// <summary>Matches an anonymous function of the shape <c>(x) => y</c>.</summary>
+    public static bool MatchAnonymousUnaryFunctionWithSingleReturn(
+        IAnonymousFunctionOperation anonymousFunction,
+        [NotNullWhen(true)] out IReturnOperation? returnOperation)
+        => MatchAnonymousFunctionWithSingleReturn(anonymousFunction, out returnOperation)
+            && anonymousFunction.Symbol.Parameters.Length == 1;
+
+    /// <summary>Matches an anonymous function of the shape <c>(...) => y</c>.</summary>
+    public static bool MatchAnonymousFunctionWithSingleReturn(
+        IAnonymousFunctionOperation anonymousFunction,
+        [NotNullWhen(true)] out IReturnOperation? returnOperation)
+    {
+        returnOperation = null;
+        return anonymousFunction.Body.Operations.Length == 1
+           && anonymousFunction.Body.Operations[0] is IReturnOperation returnOperation_
```

Comment:
Da findest du sicher einen namen ohne _

Reply:
Fixed in [11b92b6](https://github.com/polyadic/funcky/pull/680/commits/11b92b6778a997a3ad921b6a812416421fb4afd7)